.github/workflows/ci.yml

```diff
@@ -87,8 +87,13 @@ jobs:
       with:
         node-version: ${{ env.NODE_VERSION }}
 
-    - name: Run tests
-      run: EFFEKT_VALGRIND=1 EFFEKT_DEBUG=1 sbt clean test
+    - name: Run tests with coverage
+      run: EFFEKT_VALGRIND=1 EFFEKT_DEBUG=1 sbt clean coverage test coverageReport
```

Comment:
As a downside, this could make the tests _much_ slower :'(
In that case, we could also running tests with coverage separately to reduce the turnaround.